Tools/Release/buildRelease.sh

```diff
@@ -43,6 +43,14 @@ git clone $REPO_URL --depth=1 --branch $TAG
 cd $REPO_NAME
 
 
+# Fastlane update
```

Comment:
We do not need to do it twice in the script. The first `Fastlane update` block must be removed.